native/packages/react-native-bpk-theming/readme.md

```diff
@@ -16,17 +16,33 @@ import { View } from 'react-native';
 import BpkThemeProvider from 'react-native-bpk-theming';
 
 import BpkButton from 'react-native-bpk-component-button';
-import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
+
+import { translationHelper } from 'translations';
 
 const theme = {
-  color: TOKENS.colorBlue500
+  contentColor: '#2d244c',
+  backgroundColor: '#fff',
+  brandColors: {
+    gradientStart: '#fce134',
+    gradientEnd: '#f8c42d',
+  },
+};
+
+const themeAttributes = {
+  buttonPrimaryGradientStartColor: theme.brandColors.gradientStart,
```

Comment:
Interesting ... how come you show the layer of indirection here? Don't you think we should keep it simple for the docs?

Reply:
Well, I looked at our other examples and it looked like we tend to prefer more 'true to life' examples, for example in the readme for native buttons we import and make use of a fake `translationHelper`. So that was my rationale really.

Having said that I personally prefer code samples to show the minimum required, so I'm happy to change this and keep it simple.